contracts/governance/KyberVotingPowerStrategy.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import {SafeMath} from '@openzeppelin/contracts/math/SafeMath.sol';
+
+import {IVotingPowerStrategy} from '../interfaces/IVotingPowerStrategy.sol';
+import {IKyberGovernance} from '../interfaces/IKyberGovernance.sol';
+import {IKyberStaking} from '../interfaces/IKyberStaking.sol';
+import {EpochUtils} from '../misc/EpochUtils.sol';
+
+/**
+ * @title Governance Strategy contract
+ * @dev Smart contract containing logic to measure users' relative power to propose and vote.
+ * @author Aave
+ **/
+contract KyberVotingPowerStrategy is IVotingPowerStrategy, EpochUtils {
+  using SafeMath for uint256;
+
+  uint256 public constant MAX_PROPOSAL_PER_EPOCH = 10;
+  IKyberStaking public immutable staking;
+  IKyberGovernance public immutable governance;
+
+  mapping(uint256 => uint256[]) internal epochProposals;
+
+  /**
+   * @dev Constructor, register tokens used for Voting and Proposition Powers.
+   * @param _staking The address of the knc staking contract.
+   **/
+  constructor(IKyberGovernance _governance, IKyberStaking _staking) {
+    staking = _staking;
+    governance = _governance;
+
+    /// init epochutils
+    epochPeriodInSeconds = _staking.epochPeriodInSeconds();
+    firstEpochStartTimestamp = _staking.firstEpochStartTimestamp();
+  }
+
+  /// @dev endTimestamp: furture usage
+  function handleProposalCreation(
+    uint256 proposalID,
+    uint256 startTimestamp,
```

Comment:
should change `proposalID` -> `proposalId` and `startTimestamp` -> `startTime` for consistency

Reply:
change as request